install-librephotos.sh

```diff
@@ -131,6 +131,8 @@ curl -SL https://github.com/LibrePhotos/librephotos-docker/releases/download/0.1
 curl -SL https://github.com/LibrePhotos/librephotos-docker/releases/download/0.1/im2txt.tar.gz | tar -zxC $BASE_DATA/data_models/
 curl -SL https://github.com/LibrePhotos/librephotos-docker/releases/download/0.1/clip-embeddings.tar.gz | tar -zxC $BASE_DATA/data_models/
 mkdir -p ~/.cache/torch/hub/checkpoints/
+# Make sure pip can do it's work
+rm /usr/lib/python3.11/EXTERNALLY-MANAGED
```

Comment:
maybe use rm -f.

Reply:
I also had the issue detailed in #62  - At least for me, this fix alone didn't work. I needed to remove the EXTERNALLY-MANAGED file PRIOR to running the install.

Reply:
I understand."EXTERNALLY-MANAGED" does not exist for me and "rm -f /usr/lib/python3.11/EXTERNALLY-MANAGED" avoids the shell script to abort.

Reply:
Don't think that's needed. To install packages for python 3.11 we are using `--break-system-packages` pip flag. See [backend/Dockerfile](https://github.com/LibrePhotos/librephotos-docker/blob/main/backend/Dockerfile#L13).